src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3194,8 +3194,114 @@ def test_sql_managed_db_short_retention(self, resource_group, resource_group_loc
                      self.check('retentionDays', '{retention_days_dec}')])
 
 
-class SqlManagedInstanceRestoreDeletedDbScenarioTest(ScenarioTest):
+class SqlManagedInstanceDbLongTermRetentionScenarioTest(ScenarioTest):
+    @record_only()
```

Comment:
Why do you make it record_only() here? Is there any existing resource reference?